### PR TITLE
test(git): Decouple 'gitfs/fs' and 'gitfs/git' in tests

### PR DESCRIPTION
**Pull request by zharinov**
This should make further refactoring way easier.

Ref: #6617

### CODE REVIEW

File: lib/manager/npm/extract/index.spec.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import fs from 'fs';
+import { readFileSync } from 'fs';
 import path from 'path';
 import { getConfig } from '../../../config/defaults';
-import * as _gitfs from '../../../util/gitfs/fs';
+import * as _fs from '../../../util/gitfs/fs';
```
Comment (viceice) [inline lib/manager/npm/extract/index.spec.ts:4]: Can we import this from `test/util` too here

File: lib/platform/azure/index.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import is from '@sindresorhus/is';
 import { REPOSITORY_DISABLED } from '../../constants/error-messages';
 import { BranchStatus } from '../../types';
-import * as _gitfs from '../../util/gitfs';
+import * as _git from '../../util/gitfs/git';
```
Comment (viceice) [inline lib/platform/azure/index.spec.ts:4]: import from `test/util` ?

**Review by viceice — COMMENTED** [review]
`gitfs/git` seems to be ugly naming, we should rename it in a later pr.

File: lib/platform/azure/index.spec.ts (hunk 2 above)
Comment (zharinov) [reply lib/platform/azure/index.spec.ts:4]: Unfortunately not so easy, these two namespaces (`platform/*` and `npm/extract`) brought me massive headaches when I tried

Comment (viceice) [reply lib/platform/azure/index.spec.ts:4]: So maybe we should refactor these tests to use regular mocking before this pr?

**Comment by rarkins** [comment]
We were debating it on slack. Got a better idea? Are you ok with combining or should we put them separate under util?

File: lib/platform/azure/index.spec.ts (hunk 2 above)
Comment (zharinov) [reply lib/platform/azure/index.spec.ts:4]: Let's do this later, I already have a pool of refactorings which I planned to make once we get rid of `extra-fs` usage outside of `gitfs`

**Comment by rarkins** [comment]
Let's keep it moving..

**Pull request merged**